testing/api/test_utils.py

```diff
@@ -7,6 +7,23 @@
 from spinalcordtoolbox import utils
 
 
+def test_strip_py():
+    """
+    Test that utils.strip_py() correctly removes '.py' suffix if present.
+    """
+    # The happy path
+    assert utils.strip_py("basename.py") == "basename"
+    assert utils.strip_py("basename") == "basename"
+
+    # A short string
+    assert utils.strip_py("a") == "a"
+
+    # Permutations and prefixes should not be removed
+    assert utils.strip_py("ppp.yyy") == "ppp.yyy"
+    assert utils.strip_py("no_dot_py") == "no_dot_py"
+    assert utils.strip_py(".py_prefix") == ".py_prefix"
+
+
```

Comment:
I appreciate your commitment to writing thorough tests! I'm wondering if this is absolutely necessary, though? 

This is a very straightforward utility function, composed of basic built-in Python logic. So, once we run this test once to demonstrate that the function works as intended, I'm wondering if the test is all that useful anymore as something we run on every PR. It's a very isolated function, and one that isn't likely to change...

I think I'd almost prefer having one fewer test to have to maintain? Though, I'm probably nitpicking here, aha. If anything, my comment here is just me trying to reason about like, "What our motivation is to write tests? What makes a useful test?" Etc. :thinking:

Reply:
You're right, I'll get rid of this test. I like your reasoning. In the spirit of sharing thoughts, my motivations were pretty weak:
- Partly this was an exercise in just finding our existing test suite and adding to it.
- Partly this was documentation of cases which failed with the previous code, especially since that code looked quite natural.
- And finally, since we just turned on codecov I kind of just wanted to see the number go up once :stuck_out_tongue_winking_eye: